wicket-core/src/main/java/org/apache/wicket/markup/html/debug/pageview.css

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Why don't you use the same camel case as for class and it's other resources. I.e. PageView.css

Reply:
js/css files usually have low-case names

Reply:
Yeah. It makes sense for resources.